docs/readme.md

```diff
@@ -67,6 +67,11 @@ Support
 
 Release Notes
 -------------
+### 10.1.1
+- Changes
+    - Crashlytics (Android): Fixed an [issue](https://github.com/firebase/firebase-android-sdk/issues/4313)
```

Comment:
Can you add:
- General (Android): Update to Firebase Android BoM version 31.0.3.